config/README.rst

```diff
@@ -45,7 +46,7 @@ Each directory contains the following files:
 
   - This file is copied to `.gitignore`.
 
-* MANIFEST.in
+* MANIFEST.in.jj2
```

Comment:
This is the first time I see `.jj2`.  I've seen `.j2` before, for Jinja templates.

Reply:
I am going to fix it. I've rarely used Jinja2 templates before.